apps/datahub/src/assets/i18n/en.json

```diff
@@ -0,0 +1,47 @@
+{
+  "datafeeder.month.april": "",
+  "datafeeder.month.august": "",
+  "datafeeder.month.december": "",
+  "datafeeder.month.february": "",
+  "datafeeder.month.january": "",
+  "datafeeder.month.july": "",
+  "datafeeder.month.june": "",
+  "datafeeder.month.march": "",
+  "datafeeder.month.may": "",
+  "datafeeder.month.november": "",
+  "datafeeder.month.october": "",
+  "datafeeder.month.september": "",
+  "datafeeder.wizardSummarize.createdAt": "",
+  "datafeeder.wizardSummarize.scale": "",
+  "dropFile": "",
+  "facets.block.title.OrgForResource": "",
+  "facets.block.title.availableInServices": "",
+  "facets.block.title.cl_hierarchyLevel.key": "",
+  "facets.block.title.cl_maintenanceAndUpdateFrequency.key": "",
+  "facets.block.title.cl_spatialRepresentationType.key": "",
+  "facets.block.title.cl_status.key": "",
+  "facets.block.title.creationYearForResource": "",
+  "facets.block.title.resolutionScaleDenominator": "",
+  "facets.block.title.tag": "",
+  "facets.block.title.tag.default": "",
+  "facets.block.title.th_regions_tree.default": "",
+  "mdview.tab.data": "Data",
+  "mdview.tab.exports": "Exports",
+  "mdview.tab.info": "Information",
+  "mdview.tab.map": "Map",
+  "next": "",
+  "previous": "",
+  "record.action.download": "",
+  "record.action.view": "",
+  "record.more.details": "",
+  "records": "",
+  "results.layout.selectOne": "",
+  "results.records.hits.found": "",
+  "results.records.hits.found=0.help": "",
+  "results.sortBy.dateStamp": "",
+  "results.sortBy.popularity": "",
+  "results.sortBy.relevancy": "",
+  "search.field.any.placeholder": "",
+  "search.field.sortBy": "",
+  "search.loading": ""
+}
```

Comment:
These files are not needed any more; to extract keys, run `npm run i18n:extract`, the keys will be added to the files in the `translations` directory